docs/upgrading_to_v1.0.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading to v1.0
+The v1.0 release of *cloud-spanner* is a backwards incompatible
+release.
+
+### Google Cloud Platform Provider upgrade
+The Terraform Cloud Spanner Module now requires version 6.1 or higher of the Google Cloud Platform Providers.
+
+```diff
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+-      version = ">= 3.53, < 7"
++      version = ">= 6.1, < 7"
+    }
+}
+```
+
+### Backup Module Output Changes
```

Comment:
Looks good - Do we need to address removed variables `backup_schedule` and `backup_schedule_region`?

Reply:
Added